client/src/components/subscription/subscriptionsList.js

```diff
@@ -72,29 +92,39 @@ class SubscriptionsList extends React.Component {
         <div key={subscriptionUuid} className="m-2">
           {!editingSubscription &&
             <div className="d-flex align-items-center flex-wrap">
-              <Subscription details={subscription} handleEdit={() => this.setState({ editingSubscription: subscriptionUuid})} handleDelete={() => this.handleDelete(subscriptionUuid)} />
+              <Subscription details={subscription} handleEdit={() => this.setState({ editingSubscription: subscriptionUuid, addingSubscription: false })} handleDelete={() => this.handleDelete(subscriptionUuid)} />
             </div>
           }
           {editingSubscription === subscriptionUuid && 
             <div className="card p-3 m-2 d-flex flex-wrap">
-              <UpdateSubscription updateSubscription={this.handleUpdate} currentSubscriptions={subscriptions} prevSubscription={subscription} />
+              <UpdateSubscription updateSubscription={this.handleUpdate} toggleLoadingState={this.toggleLoadingState} currentSubscriptions={subscriptions} prevSubscription={subscription} />
               <button onClick={() => this.setState({ editingSubscription: null })} className="btn btn-link my-2" type="button">Cancel</button>
             </div>
             }
         </div>
       );
     });
 
+    const addSubScriptionTemplates = addingSubscription
```

Comment:
change addSubScriptionTemplates to addSubscriptionTemplates

```suggestion
    const addSubscriptionTemplates = addingSubscription
```